src/rustdoc-json-types/lib.rs

```diff
@@ -927,6 +927,16 @@ pub enum TraitBoundModifier {
     MaybeConst,
 }
 
+/// One precise capturing argument.
```

Comment:
NIT: This should link to [the docs](https://doc.rust-lang.org/reference/types/impl-trait.html#precise-capturing) on precise capturing.